cppapi/server/dserversignal.h

```diff
@@ -150,7 +150,7 @@ protected :
 	}
 #endif
 
-#if (defined __GLIBC__ || defined __darwin__ || defined __freebsd__)
+#if (defined __GLIBC__ || defined __darwin__ || defined __FreeBSD__ || defined __OpenBSD__)
 	static inline bool auth_signal(TANGO_UNUSED(long s)) {return true;}
```

Comment:
I think we should delete this function since it does not anything useful and no other definition is provided for different OSs. We could then remove this dead code block from `dserversignal.cpp`:

```cpp
	if (auth_signal(signo) == false)
	{
		TangoSys_OMemStream o;
		o << "Signal " << sig_name[signo] << "is not authorized with your OS" << std::ends;
		Except::throw_exception((const char *)API_SignalOutOfRange,
				      o.str(),
				      (const char *)"DServerSignal::register_dev_signal");
	}
```

Reply:
Thanks @mliszcz
Suggestion implemented in 918e64594a32acd31bcc6633e621c069cebfe453